docs/starter-nextjs-supabase-writeup.md

```diff
@@ -240,6 +240,8 @@ npm run dev
 
 Your backend application will now be accessible at [http://localhost:3000](http://localhost:3000). Use this URL to interact with the API endpoints and verify the setup.
```

Comment:
Thanks to Daytona magic and ports forwarding your backend...

Reply:
I've now updated this part with - **Thanks to Daytona's magic and ports forwarding, you can instantly access and interact with your API endpoints to verify the setup.** please check the latest commit.